src/rpc/blockchain.cpp

```diff
@@ -1180,6 +1180,14 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     obj.push_back(Pair("verificationprogress",  GuessVerificationProgress(Params().TxData(), chainActive.Tip())));
     obj.push_back(Pair("chainwork",             chainActive.Tip()->nChainWork.GetHex()));
     obj.push_back(Pair("pruned",                fPruneMode));
+    if (fPruneMode)
```

Comment:
Since this is touched, update as per developer notes? `{` in this line.

Reply:
Done in #11367